subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -1069,6 +1183,31 @@ static bool send_request(enum http_method method,
 		}
 
 		break;
+
+	case HAWKBIT_DOWNLOAD_PART:
+#ifdef CONFIG_HAWKBIT_DOWNLOAD_PART
+		hb_context.http_req.optional_headers_cb = header_cb;
+		hb_context.code_status = HAWKBIT_OK;
+
+		do {
+			hb_context.dl.http_content_size = 0;
+			ret = http_client_req(hb_context.sock, &hb_context.http_req,
+					      HAWKBIT_RECV_TIMEOUT, "HAWKBIT_DOWNLOAD_PART");
+			if (ret < 0) {
+				LOG_ERR("Unable to send HTTP request (HAWKBIT_DOWNLOAD_PART): %d",
+					ret);
+				return false;
+			}
+			if (hb_context.code_status == HAWKBIT_NETWORKING_ERROR) {
+				return false;
+			}
+			hb_context.dl.parts_downloaded++;
```

Comment:
parts_downloaded variable is incremented. but it is not consumed anywhere in the code

Reply:
It has been removed